project/AkkaBuild.scala

```diff
@@ -286,4 +288,24 @@ object AkkaBuild {
   }
 
   def majorMinor(version: String): Option[String] = """\d+\.\d+""".r.findFirstIn(version)
+
+  // So we can `sbt "+~ 2.13 clean compile"`
```

Comment:
What's the difference here to `++`?

Reply:
Two differences:
* `++` also requires the patch version
* When subprojects need to be excluded, `++` needs to be specified for each command

So the `++` equivalent of the above example is `sbt "++ 2.13.7 clean" "++ 2.13.7 compile"

Reply:
Btw. the syntax for multiple commands uses `;`, so you could probably also write, `++ 2.13.7 ;clean;compile`.

Reply:
> That's great, can you add that as a comment? Maybe we should also create an issue with sbt to maybe get that as an upstream feature eventually.

https://discuss.lightbend.com/t/feature-request-fuzzy-switch/9288

Reply:
> Interestingly that doesn't seem to work for me (with `sbt "++ 3.1.1-RC1 clean;compile"` the non-scala3 modules get compiled anyway, leading to trouble)

Yep, indeed that aspect seems to be different.